KPI/views/firefox_desktop_usage_2021.view.lkml

```diff
@@ -167,55 +171,64 @@ derived_table: {
   }
 
   measure: delta_from_forecast {
+    label: "Cdou: Relative Delta from Forecast"
     type: number
     value_format: "0.000%"
     sql: (${recent_cdou} / ${prediction.recent_cdou_forecast} ) - 1 ;;
   }
 
   measure: delta_from_target {
+    label: "Cdou: Relative Delta from Target"
     type: number
     value_format: "0.000%"
     sql: (${recent_cdou} / (${prediction.recent_cdou_target}) ) - 1 ;;
   }
 
   measure: delta_from_target_count {
+    label: "Cdou: Absolute Delta from Target"
     type: number
     value_format: "#,##0"
     sql: ${cdou} - ${prediction.cdou_target} ;;
   }
 
   measure: delta_from_forecast_count {
+    label: "Cdou: Absolute Delta from Forecast"
     type: number
     value_format: "#,##0"
     sql: ${cdou} - ${prediction.cdou_forecast}  ;;
   }
 
   measure: delta_from_forecast_new_profiles {
+    hidden: yes
     type: number
     value_format: "0.000%"
     sql: (${recent_new_profiles_cumulative} / ${prediction.recent_cum_new_profiles_forecast} ) - 1 ;;
   }
 
   measure: delta_from_target_new_profiles {
+    hidden: yes
     type: number
     value_format: "0.000%"
     sql: (${recent_new_profiles_cumulative} / (${prediction.recent_cum_new_profiles_target}) ) - 1 ;;
   }
 
   measure: delta_from_forecast_new_profiles_count {
+    label: "New Profiles: Absolute Delta from Forecast"
```

Comment:
I also clarified a bunch of measures using `label`. I think the explore now looks cleaner and maybe a bit less intimidating.